app/views/articles/index.html.haml

```diff
@@ -15,3 +15,12 @@
 
 %div.thumb
   = render :partial => 'card'
+
+:javascript
+  jQuery( document ).ready(function( $ ) {
```

Comment:
all javascript should be included in the assets, to keep it unobtrusive. Refer:
http://stackoverflow.com/questions/6287440/rails-3-good-rule-of-thumb-for-where-to-put-javascript